widgetastic_manageiq/__init__.py

```diff
@@ -1816,6 +1816,50 @@ def _view_buttons(self):
             yield self.compressed_button
 
 
+class ReportToolBarViewSelector(View):
+    """ represents toolbar's view selector control
+        it is present on pages with items like Infra or Cloud Providers pages
+
+    .. code-block:: python
+
+        view_selector = View.nested(ItemsToolBarViewSelector)
+
+        view_selector.select('Tile View')
+        view_selector.selected
+    """
```

Comment:
**REQUIRED**
Seems like copy paste from `ItemsToolBarViewSelector` 😄. Maybe it's better to inherit `ItemsToolBarViewSelector` and redefine ROOT and buttons?

Reply:
I will fix the comment, but I think I'll need to pull out into a common View, which I will do in another PR, but thank you for this